aries-site/.env.local

```diff
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_FEEDBACK_APP_API_TOKEN=acLCL6dZED3y8nZhqPSq1R6rCTFh1adqrYEXPdfY
```

Comment:
Actually, I think we should be excluding this file. I don't think we want our token checked into the repo. It should be added to our Netlify config.